sorc/chgres_cube.fd/input_data.F90

```diff
@@ -48,7 +48,7 @@ module input_data
 
  private
 
- public :: check_soilt
+ public :: check_soilt, check_cnwat
```

Comment:
These need doxygen documentation.

Reply:
I'm not sure what you're asking for here. The subroutines themselves have documentation at the top of each one. Do you just want me to write out to the side that these are subroutines contained in the module?

Reply:
Sorry I thought they were module variables...